ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/manager/EthScheduler.java

```diff
@@ -45,9 +47,9 @@
   private static final Logger LOG = LogManager.getLogger();
 
   private final Duration defaultTimeout = Duration.ofSeconds(5);
-
   private final AtomicBoolean stopped = new AtomicBoolean(false);
   private final CountDownLatch shutdown = new CountDownLatch(1);
+  private static final int txWorkerCapacity = 1000000;
```

Comment:
nit: constants should use all uppercase names
```suggestion
  private static final int TX_WORKER_CAPACITY = 1000000;
```